cloudflare/resource_cloudflare_teams_rules.go

```diff
@@ -0,0 +1,395 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+var teamsRuleActions = []string{
+	"allow",
+	"block",
+	"safesearch",
+	"ytrestricted",
+	"on",
+	"off",
+	"scan",
+	"noscan",
+	"isolate",
+	"noisolate",
+	"override",
+	"l4_override",
+}
```

Comment:
this should probably live in `cloudflare-go` similar to Rulesets with a helper method to expose them as a string list.

- https://github.com/cloudflare/cloudflare-go/blob/master/rulesets.go#L20-L30
- https://github.com/cloudflare/cloudflare-go/blob/master/rulesets.go#L69-L83

Reply:
This makes sense, but would require waiting for another cloudflare-go release next month. Can we merge it for now as-is and ill be sure to make a ticket and put up a PR to take care of it later?

Reply:
sure, sounds like a plan 👍

Reply:
https://github.com/cloudflare/cloudflare-go/pull/702